### PR TITLE
Use the API to determine if the Taobao platform supports websocket multiple instances.

### PR DESCRIPTION
**Pull request by bofeng-song**
Re: #

### Changelog

*

-------

### Continuous Integration

This pull request:

* [ ] needs automatic test cases check.
  > Manual trigger with `@cocos-robot run test cases` afterward.
* [ ] does not change any runtime related code or build configuration
  > If any reviewer thinks the CI checks are needed, please uncheck this option, then close and reopen the issue.

-------

### Compatibility Check

This pull request:

* [ ] changes public API, and have ensured backward compatibility with [deprecated features](https://github.com/cocos/cocos-engine/blob/v3.5.0/docs/contribution/deprecated-features.md).
* [ ] affects platform compatibility, e.g. system version, browser version, platform sdk version, platform toolchain, language version, hardware compatibility etc.
* [ ] affects file structure of the build package or build configuration which requires user project upgrade.
* [ ] **introduces breaking changes**, please list all changes, affected features and the scope of violation.

### CODE REVIEW

File: platforms/minigame/platforms/taobao-mini-game/wrapper/builtin/window.js [hunk 1]
```diff
@@ -13,23 +13,10 @@ export cancelAnimationFrame from './cancelAnimationFrame'
...
-        if (curVersionPart < supportedVersionPart) return -1;
-    }
-    return 0;
-}
-
 import MultiWebSocket from './MultiWebSocket'
 import WebSocket from './WebSocket'
 const systemInfo = my.getSystemInfoSync();
-if (compareVersions(systemInfo.version, '10.35.30') >= 0) {
+if (typeof my !== 'undefined' && my && my.canIUse && my.canIUse('connectSocket.object.multiple')) {
```
Comment (minggo) [inline platforms/minigame/platforms/taobao-mini-game/wrapper/builtin/window.js:19]: Is typeof my !== 'undefined' needed?

Comment (bofeng-song) [reply platforms/minigame/platforms/taobao-mini-game/wrapper/builtin/window.js:19]: image

Comment (minggo) [reply platforms/minigame/platforms/taobao-mini-game/wrapper/builtin/window.js:19]: Can just use like
```
if (my?.canIUse?.('connectSocket.object.multiple')) {
} else {
}
```

Comment (bofeng-song) [reply platforms/minigame/platforms/taobao-mini-game/wrapper/builtin/window.js:19]: report error on Chrome
image

**Review by minggo — APPROVED** [review]

**Pull request merged**